### ISSUE TITLE
white/empty pop-up window

### ISSUE THREAD

**Issue by enterz** [Mafrans/StadiaPlus#168]
**Problem description**
I've installed the plugin into Chrome and when I click the extension button, I only see an empty pop-up.
![image](https://user-images.githubusercontent.com/4612031/104342117-c8a8e080-54fa-11eb-90ee-a468d47d3ef9.png)

**Troubleshooting**
When I 'inspect' the popup in Chrome I can see an error in the console:

```
[Language.ts:70]
Uncaught (in promise) TypeError: Cannot read property 'tag' of undefined
    at Function.<anonymous> (Language.ts:70)
    at Generator.next (<anonymous>)
    at r (popup.js:1)
```

When I open the Language.ts on line 70 the error occurs on a Logging line:

![image](https://user-images.githubusercontent.com/4612031/104342752-761bf400-54fb-11eb-93f6-37aae48c77e8.png)

**Version information**
Chrome information:
Version 87.0.4280.88 (Official Build) (64-bit)
Linux, PopOS 20.04 (Also tried on Windows 10, up-to-date, same result)

StadiaPlus extension: 2.5.7
Stadia: Pro account

**Comment by frietbakker** [comment]
I have this too.

**Comment by Mafrans** [comment]
This should be fixed as of 2.5.8
